src/pages/newCommunity/NewCommunity.jsx

```diff
@@ -0,0 +1,115 @@
+            <div className="ncTextBox">
+              <TextField
+                required
+                fullWidth
+                id="outlined-basic"
+                label="Community Name"
+                variant="outlined"
+              />
+            </div>
+          </Grid>
+
+          <Grid item xs={12}>
+            <div className="ncTextBox">
+              <TextField
+                multiline
+                required
+                fullWidth
+                maxRows={15}
+                minRows={15}
+                id="outlined-basic"
+                label="Community Description"
+                variant="outlined"
+              />
+            </div>
+          </Grid>
+          <Grid item xs={12}>
+            <div className="npCarouselWrapper">
+              <Carousel dynamicHeight showThumbs={false} autoPlay infiniteLoop>
+                {images.map((image) => {
+                  return (
+                    <Box>
+                      <img alt="uoaimage" src={image} />
+                      <Button
+                        className="legend"
+                        color="primary"
+                        style={{ top: "20px", bottom: "auto" }}
+                        onClick={() => {
+                          setImages(
+                            images.filter((imageMain) => imageMain !== image)
+                          );
+                        }}
+                      >
+                        Delete
+                      </Button>
+                    </Box>
+                  );
+                })}
+              </Carousel>
+            </div>
+          </Grid>
+          <Grid item xs={6}>
+            <div className="ncUploadButton">
+              <Button
+                label="Upload media"
+                variant="outlined"
+                component="label"
+                endIcon={<AddPhotoAlternateIcon />}
+              >
+                Upload Background Image
+                <input
+                  type="file"
+                  hidden
+                  onChange={(evt) => {
+                    const file = evt.target.files[0];
+
+                    var imageType = /image.*/;
+
+                    if (file && file.type.match(imageType)) {
+                      var reader = new FileReader();
+                      reader.onload = function (e) {
+                        const imageString = reader.result;
+                        const newImagesArray = [...images, imageString];
+                        setImages(newImagesArray);
+                      };
+
+                      reader.readAsDataURL(file);
+                    }
+                  }}
+                />
+              </Button>
+            </div>
+          </Grid>
+
+          <Grid item xs={6}>
+            <div className="ncSubmitButton">
+              <Button variant="contained">Create</Button>
+            </div>
+          </Grid>
+        </Grid>
+      </form>
+
+      <div></div>
```

Comment:
There seems to be a redundant div tag here; if it's not for a specific purpose, I think it should be removed.

```suggestion
```

Reply:
thank you for your suggestion, i deleted that